static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js

```diff
@@ -244,14 +205,14 @@ define([
             },
 
             {
-                id: 'just-contribute-fake',
+                id: 'just-supporter',
 
                 test: function () {
-                    var ctaType = cta.justContribute;
-                    var message = messages.fake;
+                    var ctaType = cta.justSupporter;
+                    var message = messages.control;
                     var component = $.create(template(contributionsEpicEqualButtons, {
-                        linkUrl1: ctaType.url1 + message.intcmp + '_contribute',
-                        linkUrl2: ctaType.url2 + message.intcmp + '_contribute',
+                        linkUrl1: ctaType.url1 + message.intcmp + '_supporter',
+                        linkUrl2: ctaType.url2 + message.intcmp + '_supporter',
```

Comment:
Also, the ctaType.url1 and ctaType.url2 for just supporter and just contribute can lose the "_backup" now, as they are no longer being used as backups